static/src/js/zustand/types.ts

```diff
@@ -866,6 +866,34 @@ export type UiSlice = {
   }
 }
 
+// Holds project panel state this is: panel.group.section
```

Comment:
This is the wrong comment syntax for TS. vscode will show you the comment when you hover over something that is the type if you use the `/** Comment */` syntax